src/components/EthExchanges/use-eth-exchanges.ts

```diff
@@ -492,16 +493,15 @@ export const useEthExchanges = () => {
   }
 
   // Add `value` & `label` for Select component
-  const exchangesByCountry: Array<ExchangeByCountry> =
-    data.exchangesByCountry.nodes
-      .map((node) => {
-        return {
-          value: node.country,
-          label: node.country,
-          exchanges: node,
-        }
-      })
-      .sort((a, b) => a.value.localeCompare(b.value))
+  const exchangesByCountry: Array<ExchangeByCountry> = shuffle(
```

Comment:
I think we should revert this since it shuffles the countries in the list. I think this part should remain alphabetical. But, everything else is good and shuffles the exchanges and wallets which I think is good. 

![Screen Shot 2023-06-29 at 8 46 30 AM](https://github.com/ethereum/ethereum-org-website/assets/15589226/28148e92-d278-4a76-8bdd-f8de5d888b22)

Reply:
@wackerow 👆

Reply:
Pushed up this change after talking to @wackerow